source/content/guides/drupal-9-from-dist.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Create a Drupal 9 Site Using a Drupal Distribution
+description: Learn how to create a site using a Drupal 9 distribution.
+cms: "Drupal"
+categories: [get-started]
+tags: [site]
+type: guide
+permalink: docs/guides/drupal-9-from-dist
+contributors: [wordsmither]
+date: 07/06/2022
+---
+
+Distributions are pre-made packages that you can use to simplify creating and setting up a Drupal website. Drupal distributions are exceptionally helpful if you want to create a website, but don't want to build it from scratch. 
+
+Start by looking at the documentation provided with the community distribution; then continue with the following steps.
+
+## Prepare
+
+1. <Partial file="drupal-9/prepare-local-environment-no-clone-no-alias.md" />
+
+## Create an Empty Upstream
```

Comment:
Create an Empty Upstream site (or site based on empty upstream) (word site is the key here)